lib/loader.js

```diff
@@ -114,12 +119,33 @@ Loader = function () {
           }
           require(path.join(dirname, filePath));
         }
+        if (fs.lstatSync(path.join(dirname, filePath)).isDirectory()) {
+          self.loadDirectory(path.join(dirname, filePath))
+        }
       });
       return true;
     }
 
     return false;
   };
+
+  this.loadTasks = function (library) {
+    var modulePath = _isRelative(library) ? path.resolve(process.cwd(), library) : path.resolve(process.cwd(), 'node_modules', library)
```

Comment:
Ternaries like this are really hard to read when they get this long. Suggest either saving some of these values in local variables, or just making it a simple if/else.